content/extending-ui/extending-chat-widget/rich-messages/index.mdx

```diff
@@ -0,0 +1,267 @@
+---
+weight: 31
+category: "extending-ui"
+subcategory: "extending-chat-widget"
+title: "Rich Messages"
+tagline: "Enrich your messages."
+desc: "Engage better with your Customers using interactive and visually appealing Rich Messages."
+---
+
+# Introduction
+[Rich Messages](https://www.livechatinc.com/help/rich-messages/) are more than just plain text. These are interactive blocks, which with the use of images, buttons and text allow your Customers to get more details about your business, quickly send responses to your messages, make phone calls or even open small apps called [Moments](/extending-ui/extending-chat-widget/chat-widget-moments/).
+
+![LiveChat Rich Messages Placeholder](images/livechat-rich-messages.jpg)
+
+Rich Messages are also the core feature of our **ChatBot** integration. If you're not familiar with ChatBot, [this article](https://www.livechatinc.com/help/chatbots-explained/) will come in handy.
+
+# Rich Messages overview
+Rich Messages are built with three basic building blocks: text, image and buttons. Combining these together creates different opportunities. Following that, in LiveChat we have grouped them in the following categories.
+
+## Cards
+Cards are single message block, which can contain the image, title, subtitle and buttons:
```

Comment:
Imho it will look better without splitting the text to 2 single line paragraphs with an image between them. I suggest to combine the description into a single paragraph and after it, show the image.